xcode/Subconscious/Shared/Config.swift

```diff
@@ -17,6 +17,8 @@ struct Config: Equatable, Codable {
     var appTabs = false
     var addressBook = false
     var addByQRCode = true
+    var userProfile = false
+    var userProfileDetailsTable = false
```

Comment:
This is the WIP table that shows IPFS hash, Gateway URL etc. on the profile metasheet